zone/gm_commands/faction_association.cpp

```diff
@@ -0,0 +1,17 @@
+#include "../client.h"
+
+void command_faction_association(Client *c, const Seperator *sep)
+{
+	if (sep->argnum < 2 || !sep->IsNumber(1) || !sep->IsNumber(2)) {
+		c->Message(Chat::White, "Usage: #factionassociation <factionid> <amount>");
+		c->Message(Chat::White, "  Defaults to self, unless target is also a client.");
+		return;
+	}
+
+	// default to self unless target is also a client
+	auto target = c;
+	if (c->GetTarget() && c->GetTarget()->IsClient())
```

Comment:
Would really prefer if we just close these. There's a few other places of implied brackets